packages/ibm-products/src/components/Checklist/Checklist.js

```diff
@@ -78,8 +78,8 @@ export let Checklist = React.forwardRef(
     ref
   ) => {
     const [isOpen, setIsOpen] = useState(open);
-    const listContainerId = useMemo(() => uuidv4(), []);
-    const chartLabelId = useMemo(() => uuidv4(), []);
+    const listContainerId = useRef(uuidv4()).current;
+    const chartLabelId = useRef(uuidv4()).current;
```

Comment:
After a review of other developer's code, I found that they prefer `userRef` over `useMemo`.

Updated. Just trying to keep up with the cool kids.